lib/ansible/plugins/lookup/nios_next_ip.py

```diff
@@ -39,6 +39,10 @@
       description: The number of IP addresses to return
       required: false
       default: 1
+    exclude:
+      description: The number of IP addresses to return
+      required: false
+      default: []
 """
 
 EXAMPLES = """
```

Comment:
@dionben Thanks for adding the respective feature, can you please include a valid running example in the doc for the users to follow for the exclude IP.